CHANGELOG.md

```diff
@@ -11,9 +11,16 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Added
+
 - Added more environment variables (PARTICLE_API_KEY, database credentials) (CU-86dupwqgq).
-- Added script to use Particle API to call a function for all of a client's sensors (CU-86dup733k).
+- New tool: Particle Client Functions for calling particle functions for all devices of client (CU-86due80fw).
```

Comment:
it would be great if u can be more detailed with what you added (such as all the different functions and pages added) and where you added them using `insert function or file name here`

Reply:
Okay, will do 👍